Dockerfile

```diff
@@ -0,0 +1,50 @@
+FROM ubuntu:latest
+MAINTAINER Harshit Prasad
+
+# Update
+RUN apt-get update
+RUN apt-get upgrade -y
+
+# add packages
+RUN apt-get install -y git openjdk-8-jdk
+
+#install gradle required for build
+RUN add-apt-repository ppa:cwchien/gradle
+RUN apt-get update
+RUN apt-get install gradle
+
+# install apache ftp server 1.1.0
+RUN wget http://www-eu.apache.org/dist/mina/ftpserver/1.1.0/dist/apache-ftpserver-1.1.0.tar.gz
+RUN tar xfz apache-ftpserver-1.1.0.tar.gz
+
```

Comment:
configure apache ftp server
**Things to do:**

1.add these lines to apache-ftpserver-1.1.0/res/conf/users.properties
```
ftpserver.user.yacygrid.userpassword=fcd6f2c8d772db634cd9fd960f4668a0
ftpserver.user.yacygrid.homedirectory=./res/home
ftpserver.user.yacygrid.enableflag=true
ftpserver.user.yacygrid.writepermission=true
ftpserver.user.yacygrid.maxloginnumber=2000
ftpserver.user.yacygrid.maxloginperip=2000
ftpserver.user.yacygrid.idletime=300
ftpserver.user.yacygrid.uploadrate=0
ftpserver.user.yacygrid.downloadrate=0
```

you can refer https://unix.stackexchange.com/questions/77277/how-to-append-multiple-lines-to-a-file for how to add multiple lines into a file.
but I prefer write these configurations into a configfile and then
do,

`cat ftpconfigfile >> apache-ftpserver-1.1.0/res/conf/users.properties`